expts/main_run_multitask.py

```diff
@@ -54,7 +56,7 @@ def main(cfg: dict, run_name: str = "main", add_date_time: bool = True) -> None:
     if add_date_time:
         date_time_suffix = datetime.now().strftime("%d.%m.%Y_%H.%M.%S")
 
-    wandb.init(entity="multitask-gnn", project=cfg["constants"]["name"], config=cfg)
+    wandb.init(entity=cfg["constants"]["entity"], project=cfg["constants"]["name"], config=cfg)
 
```

Comment:
I think here if would be good to have a default entity value if no "entity" is set in the config, otherwise it will complain with an `KeyError`. Most of our configs actually don't have the entity.